packages/components/src/Table/Table.tsx

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import styled from 'styled-components';
+
+type ColumnsType = { key: string; title: string }[];
+type DataType = { [index: string]: string | number | JSX.Element | JSX.Element[] }[];
```

Comment:
[nits] You can use `React.ReactNode` instead of `string | number | JSX.Element | JSX.Element[]`

Reply:
Thanks, good to know!